### ISSUE TITLE
Crash when switching to a profile where the device isn't connected

### ISSUE THREAD

**Issue by LWuensch** [Belphemur/SoundSwitch#462]
[SoundSwitch-crashlog-3_6_2020.zip](https://github.com/Belphemur/SoundSwitch/files/4724566/SoundSwitch-crashlog-3_6_2020.zip)

**Comment by Belphemur** [comment]
Not related to Windows 10 2004.

One of your profile is using a non-connected device. I'll see what I can do to improve the error handling.

**Comment by LWuensch** [comment]
Hi Antoine,

 

thank you for your reply. I solved the problem. I deleted the profiles an recreated them. Now it’s working as expected.

 

It’s interesting, because I did not change anything in regard of the devices before I upgraded to 2004 …

 

Lutz Wünsch

 

Computer Consultants Lutz Wünsch

Rangenbergstr. 22

60388 Frankfurt

 

Tel.:       +49 (6109) 31227

Fax:       +49 (6109) 31228

Mobil:   +49 (171) 2062667

Mail:       <mailto:lutz.wuensch@compcons.de> lutz.wuensch@compcons.de

 



 

Von: Antoine Aflalo <notifications@github.com> 
Gesendet: Samstag, 6. Juni 2020 17:29
An: Belphemur/SoundSwitch <SoundSwitch@noreply.github.com>
Cc: Lutz Wünsch <lutz.wuensch@compcons.de>; Author <author@noreply.github.com>
Betreff: Re: [Belphemur/SoundSwitch] Problems since Windows 10 2004 Enterprise (#462)

 

Not related to Windows 10 2004.

One of your profile is using a non-connected device. I'll see what I can do to improve the error handling.
